### PR TITLE
chore: update members of Russian's localization team

### PR DESCRIPTION
**Pull request by sashasushko**
Part of https://github.com/mdn/translated-content/issues/11833

### CODE REVIEW

**Review by yin1999 — APPROVED** [review]
LGTM for l10n-zh :)

**Review by bsmth — APPROVED** [review]
Thanks @sashasushko - leaving my +1 here 👍🏻

**Review by mfuji09 — APPROVED** [review]
LGTM for `ja`!

**Review by SphinxKnight — APPROVED** [review]
Thanks @sashasushko, all clear for me.

**Comment by SphinxKnight** [comment]
Waiting a few days for the remaining reviews but will merge this anyway if nothing else happens.

File: files/fr/mdn/community/contributing/translated_content/index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Nous avons actuellement dégelé les langues suivantes :
 ### Russe (ru)
 
 - Discussions : [Matrix (#mdn-l10n-ru channel)](https://chat.mozilla.org/#/room/#mdn-l10n-ru:mozilla.org)
-- Contributeurs actuels : [armanpwnz](https://github.com/armanpwnz), [captainspring](https://github.com/captainspring), [mpstv](https://github.com/mpstv), [myshov](https://github.com/myshov), [Saionaro](https://github.com/Saionaro), [sashasushko](https://github.com/sashasushko), [lex111](https://github.com/lex111)
+- Contributeurs actuels : [captainspring](https://github.com/captainspring), [mpstv](https://github.com/mpstv), [myshov](https://github.com/myshov), [Saionaro](https://github.com/Saionaro), [sashasushko](https://github.com/sashasushko), [lex111](https://github.com/lex111)
```
Comment (queengooborg) [inline files/fr/mdn/community/contributing/translated_content/index.md:41]: It doesn't seem like `mpstv` should be in the list any longer:

```suggestion
- Contributeurs actuels : [captainspring](https://github.com/captainspring), [myshov](https://github.com/myshov), [Saionaro](https://github.com/Saionaro), [sashasushko](https://github.com/sashasushko), [lex111](https://github.com/lex111)
```

**Review by queengooborg — CHANGES_REQUESTED** [review]
Gonna leave a +1 for Spanish, though this change is straightforward enough anyways!

I'm going to request changes for French, though -- it looks like there's a member still listed there that probably shouldn't be!

**Review by Graywolf9 — APPROVED** [review]
LGTM for `es` 👍

File: files/fr/mdn/community/contributing/translated_content/index.md (hunk 1 above)
Comment (yin1999) [reply files/fr/mdn/community/contributing/translated_content/index.md:41]: It seems that the GitHub account [`captainspring`](https://github.com/captainspring) is deleted, hi @sashasushko should we also remove this?

Comment (queengooborg) [reply files/fr/mdn/community/contributing/translated_content/index.md:41]: Looks like that account has changed their username, see #13014!

Comment (yin1999) [reply files/fr/mdn/community/contributing/translated_content/index.md:41]: Yeah, let's also sync this :)

**Review by queengooborg — APPROVED** [review]
I went ahead and applied the change, and then updated one contributor's name (`captainspring` → `yanaklose`) to match the username change!

**Pull request merged**